packages/wonder-blocks-dropdown/src/index.ts

```diff
@@ -21,4 +21,4 @@ export {
     Listbox,
 };
 
-export type {Labels, SingleSelectLabels};
+export type {LabelsValues, SingleSelectLabelsValues};
```

Comment:
The type renames will also be breaking changes since we export them. If we want to keep the rename, we'll also have to update existing imports and mention it in the changeset!

Reply:
@marcysutton any thoughts about this other rename breaking change!

Reply:
I think this one is more important to communicate the nature of the object, since it now contains values. I can add a note to the changelog!